auctions/dutch-auction/project/tests/functions/create_auction.rs

```diff
@@ -0,0 +1,172 @@
+pub mod passing {
+    use crate::utils::{
+        active_auctions_of_author, auction, auction_count, auctions_of_author, auctions_won, bid,
+        create_auction, get_contract_instance,
+    };
+    use fuels::types::{ContractId, Identity};
+
+    #[tokio::test]
+    async fn can_bid() {
+        let (instance, wallet) = get_contract_instance().await;
+
+        create_auction(
+            &instance,
+            400,
+            100,
+            2, // Block height will be 1 at start, then 2 during sending of this tx
+            5,
+            Identity::Address(wallet.address().into()),
+            ContractId::zeroed(),
+        )
+        .await;
+
+        bid(&instance, 1, 400).await;
+
+        assert_eq!(
+            auctions_won(&instance, Identity::Address(wallet.address().into())).await,
+            vec![1]
+        );
+    }
+
+    #[tokio::test]
+    async fn auction_count_increases() {
+        let (instance, wallet) = get_contract_instance().await;
+
+        // retrieving active_auctions_of_author and auctions_of_author and auction_count before
+        let auction_count_before = auction_count(&instance).await;
+        let active_auctions_of_author_before =
```

Comment:
This creation of an identity is being used repeatedly, I would wrap it into a function